### ISSUE TITLE
Add scope attributes to tables in HTML Report for 508 compliance

### ISSUE THREAD

**Issue by schrolla** [cisagov/ScubaGear#93]
They should be accessible to folks using a screen reader.  This is usually a matter of compliance with web accessibility standards:

https://www.section508.gov/develop/web-content/

Its not clear to me which milestone this needs to be aligned to, but filing it as an issue for now for tracking.

In our case it should be fairly trivial because the reports are static pure text pages using simple HTML, but we need to make sure the navigation order and such things are acceptable.
